security/Member.php

```diff
@@ -837,9 +837,9 @@ public static function currentUser() {
 		$id = Member::currentUserID();
 
 		if($id) {
-			return DataObject::get_by_id('Member', $id);
+			$member = DataObject::get_by_id('Member', $id);
```

Comment:
You could avoid this variable assignment and do `DataObject::get_by_id('Member', $id) ?: null;`